src/modules/esl-carousel/core/esl-carousel.events.ts

```diff
@@ -36,17 +44,28 @@ export class ESLCarouselSlideEvent extends Event implements ESLCarouselSlideEven
   }
 }
 
+/** {@link ESLCarouselChangeEvent} init object */
 interface ESLCarouselChangeEventInit {
-  prop: string;
-  addedSlide?: ESLCarouselSlide;
-  removedSlide?: ESLCarouselSlide;
+  /** Current {@link ESLCarousel} instance config */
+  config: ESLCarouselStaticState;
+  /** Previous {@link ESLCarousel} instance config */
+  oldConfig?: ESLCarouselStaticState;
```

Comment:
Mb previousConfig? Seems to fit better

Reply:
Just think about the similarity to the MutationObserver props, e.g. `oldValue` vs `newValue`
But let's discuss this on tomorrow's call